cdk/lib/prism.ts

```diff
@@ -122,5 +123,18 @@ export class Prism extends GuStack {
 		const cfnAsg = pattern.autoScalingGroup.node
 			.defaultChild as CfnAutoScalingGroup;
 		cfnAsg.healthCheckGracePeriod = Duration.minutes(15).toSeconds();
+
+		// A temporary security group with a fixed logical ID, replicating the one removed from GuCDK v61.5.0.
```

Comment:
This will be removed in the next PR. See https://github.com/guardian/cdk/releases/tag/v61.5.0.